app/addons/activetasks/tests/activetasks.storesSpec.js

```diff
@@ -0,0 +1,91 @@
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+define([
+  'api',
+  'addons/activetasks/views',
+  'addons/activetasks/resources',
+  'addons/activetasks/components.react',
+  'addons/activetasks/stores',
+  'addons/activetasks/tests/fakeActiveTaskResponse',
+  'react',
+  'testUtils'
+], function (FauxtonAPI, Views, ActiveTasks, Components, Stores, fakedResponse, React, testUtils) {
+  var assert = testUtils.assert;
+  var TestUtils = React.addons.TestUtils;
+  var activeTasksStore = Stores.activeTasksStore;
+  var activeTasksCollection = new ActiveTasks.AllTasks({});
+  activeTasksCollection.parse(fakedResponse);
+
+  describe('Active Tasks -- Stores', function () {
+
+    beforeEach(function () {
+      activeTasksStore.init(activeTasksCollection.table, activeTasksCollection);
+      this.clock = sinon.useFakeTimers();
+    });
+
+    afterEach(function () {
+      this.clock.restore();
+    });
+
+    describe('Active Task Stores - Polling', function () {
+      var pollingWidgetDiv, pollingWidget;
+
+      it('should poll at the correct time', function () {
```

Comment:
This test is good but you need to break it up into multiple tests. Its better to test each different time in a new test.